python/spear/enhanced_input_service.py

```diff
@@ -4,18 +4,20 @@
 
 import spear
 
+
 class EnhancedInputService():
     def __init__(self, entry_point_caller):
         self._entry_point_caller = entry_point_caller
 
     def inject_input(self, enhanced_input_subsystem, input_action, input_action_value, modifiers=[], triggers=[]):
         self._entry_point_caller.call("enhanced_input_service.inject_input",
-            enhanced_input_subsystem, input_action, spear.to_json_string(obj=input_action_value), modifiers, triggers)
+                                      enhanced_input_subsystem, input_action, spear.to_json_string(obj=input_action_value), modifiers, triggers)
```

Comment:
If the call doesn't fit on a single line, follow this convention for the other function calls too:

```
        self._entry_point_caller.call(
            "enhanced_input_service.inject_input",
            enhanced_input_subsystem,
            input_action,
            spear.to_json_string(obj=input_action_value),
            modifiers,
            triggers)
```